### PR TITLE
fix: fix SelectorResponse

### PR DESCRIPTION
**Pull request by NOBLES5E**

### CODE REVIEW

File: src/rofi_rbw/rofi_rbw.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
     from rofi_rbw.action import Action
     from rofi_rbw.clipboarder import Clipboarder
     from rofi_rbw.typer import Typer
-    from rofi_rbw.selector import Selector
+    from rofi_rbw.selector import Selector, SelectorResponse
```
Comment (fdw) [inline src/rofi_rbw/rofi_rbw.py:14]: Could you please add it to the `except` branch too?

**Review by fdw — CHANGES_REQUESTED** [review]

**Comment by fdw** [comment]
Thank you 🙂

File: src/rofi_rbw/rofi_rbw.py (hunk 1 above)
Comment (NOBLES5E) [reply src/rofi_rbw/rofi_rbw.py:14]: it's already in the except branch

**Comment by tosti007** [comment]
Sorry I messed up 😓  tested it locally without the module stuff.

File: src/rofi_rbw/rofi_rbw.py (hunk 1 above)
Comment (fdw) [reply src/rofi_rbw/rofi_rbw.py:14]: I'm sorry, I missed it :(

**Comment by fdw** [comment]
Thanks again, @NOBLES5E .

> Sorry I messed up sweat tested it locally without the module stuff.

No worries. I missed it in the review, too, and we all make mistakes - as you can see in this very PR ;)

**Pull request merged**